src/OrchardCore.Modules/Orchard.Roles/Services/RoleStore.cs

```diff
@@ -67,8 +67,6 @@ public void UpdateRoles(RolesDocument roles)
         #region IRoleStore<Role>
         public async Task<IdentityResult> CreateAsync(Role role, CancellationToken cancellationToken)
         {
-            cancellationToken.ThrowIfCancellationRequested();
```

Comment:
Same for all the other `cancellationToken.ThrowIfCancellationRequested(); ` removed lines in this file.